grouper/fe/handlers/permissions_revoke.py

```diff
@@ -7,16 +8,19 @@
 
 
 class PermissionsRevoke(GrouperHandler):
+
     def get(self, name=None, mapping_id=None):
         grantable = user_grantable_permissions(self.session, self.current_user)
-        if not grantable:
+        mapping = PermissionMap.get(self.session, id=mapping_id)
+        user_is_owner = mapping and user_is_owner_of_group(self.session, mapping.group,
+            self.current_user)
+        if not grantable and not user_is_owner:
             return self.forbidden()
 
-        mapping = PermissionMap.get(self.session, id=mapping_id)
         if not mapping:
             return self.notfound()
 
-        allowed = False
+        allowed = user_is_owner
```

Comment:
it'd be cleaner to rename `allowed` to something more descriptive and just add `user_is_owner` to the `if` that enforces our ACL